### ISSUE TITLE
Link references for citation should be yellow

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4907]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Links are not appearing yellow. PWD may not recognize they are links.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS, macOS: Safari with VoiceOver, iOS/iPadOS: Safari with VoiceOver, Android: Chrome with TalkBack, Others

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Forced colors (high contrast mode)

### How to reproduce the issue?

1. Turns on high contrast in dark mode
2. Load transcript from Web Chat Loader sample 2

### What do you expect?

All items in the link references box should be yellow.

### What actually happened?

Non-URL citations in the link references box are white.

### Do you have any screenshots or recordings to repro the issue?

![image](https://github.com/microsoft/BotFramework-WebChat/assets/1622400/752c2bb0-8a77-4236-b83b-d4aa7f89d6e7)
